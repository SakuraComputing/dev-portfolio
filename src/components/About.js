import React from 'react';


export const About = (props) => (
    <section className="about section">
        <div className="section-inner">
            <h2 className="heading">About Me</h2>
            <div className="content">
                <p>{props.about.text}</p>
            </div>
        </div>
    </section>
);

export default About;
