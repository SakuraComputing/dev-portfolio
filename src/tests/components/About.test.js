import React from 'react';
import { shallow } from 'enzyme';
import About from '../../components/About';

describe('About Section', () => {
   it('should render correctly', () => {
       const about = {
           text: "Test Text"
       };
       const wrapper = shallow(<About about={about} />);
       expect(wrapper).toMatchSnapshot();
   });
});