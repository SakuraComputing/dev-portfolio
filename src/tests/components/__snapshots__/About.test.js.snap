// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Section should render correctly 1`] = `
<section
  className="about section"
>
  <div
    className="section-inner"
  >
    <h2
      className="heading"
    >
      About Me
    </h2>
    <div
      className="content"
    >
      <p>
        Test Text
      </p>
    </div>
  </div>
</section>
`;
